### DISCUSSION TITLE
mod_redirect模块里可以使用res_code_in的条件语句吗，因为使用req_path_prefix_in可以重定向，但使用res_code_in不行？

### DISCUSSION THREAD

**Discussion by zhhOceanfly** [bfenetworks/bfe#1024] [Q&A (问题和回答)] [answered]
<img width="1125" alt="image" src="https://user-images.githubusercontent.com/15883495/157357237-58b185aa-a1e5-4238-a878-c9367db4696e.png">

**Comment by zhhOceanfly** [comment]
还有一个问题，如果mod_rewrite和mod_redirect两个都配置了，试验了下会优先走mod_redirect这个模块，是这样理解的吗

**Reply by iyangsj** [reply]
是的，mod_redirect在mod_rewrite之前执行。
你可以通过 http://localhost:<MONITOR_PORT>/monitor/module_handlers  查看模块的注册情况及执行顺序。

**Comment by iyangsj** [comment, marked as answer]
@zhhOceanfly 不可以。

因为这样的用法不对，BFE重定向模块在执行时，请求并未转发给后端。此时与响应相关的条件原语均返回false

如果真实需求是：当后端返回的响应满足某种条件（例如5xx错误），对响应进行修改替换为自定义的响应（例如改为302响应），
可以使用mod_errors 模块对将响应进行修改替换为自定义的响应。
